src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -429,11 +455,20 @@ void DlgPrefRecord::saveEncoding() {
     }
 }
 
+void DlgPrefRecord::slotToggleCueEnabled() {
+    updateCueEnabled();
+}
```

Comment:
I think these two methods are redundant. I would suggest keeping `slotToggleCueEnabled` and call it directly if you need to.

Reply:
True

Reply:
Just pushed new commit for this.